servant-client/src/Servant/Common/Req.hs

```diff
@@ -7,13 +7,17 @@
 
 module Servant.Common.Req where
 
+import Prelude hiding (any)
+
 #if !MIN_VERSION_base(4,8,0)
 import Control.Applicative
 #endif
 import Control.Exception
 import Control.Monad
 import Control.Monad.Catch (MonadThrow)
 
+import Data.Foldable (toList, any)
```

Comment:
Sort of annoying comment, but can we remove the extra space between this and the rest of the imports (and put this one in alphabetical order)?